front/app/modules/commercial/representativeness/admin/components/ChartCard/Header.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
This file has some commented-out stuff related to the representativeness index. We last minute decided that we will deprioritize it, but I will just leave the code in here for once we get to it